paymentsheet/src/test/java/com/stripe/android/link/ui/LinkScreenshotTheme.kt

```diff
@@ -0,0 +1,18 @@
+package com.stripe.android.link.ui
+
+import androidx.compose.material.Surface
+import androidx.compose.runtime.Composable
+import com.stripe.android.link.theme.DefaultLinkTheme
+import com.stripe.android.link.theme.LinkTheme
+
+@Composable
+internal fun LinkScreenshotTheme(
```

Comment:
nit: rename to `LinkScreenshotSurface` to make it clear it's adding a background
```suggestion
internal fun LinkScreenshotSurface(
```